hs_composite_resource/models.py

```diff
@@ -32,11 +32,18 @@ def set_default_logical_file(self):
         of the resource that is not already associated with a logical file. """
 
         for res_file in self.files.all():
+            # TODO: This following if block will not be needed once we migrate
+            # raster file type to HStore
```

Comment:
@pkdash I don't quite understand this comment at this point, does it mean the if block ```if not res_file.has_logical_file:``` is not needed once raster file type is migrated to HStore? If so, I cannot see why. Can you explain a bit on the difference between ```has_logical_file``` property and ```logical_file_new``` property on resource file object?